core/src/core/transaction.rs

```diff
@@ -253,9 +253,8 @@ impl TxKernel {
 	/// as a public key and checking the signature verifies with the fee as
 	/// message.
 	pub fn verify(&self) -> Result<(), Error> {
-		if (self.is_height_locked() != (self.lock_height > 0)
-		|| (self.is_coinbase()      &&  self.fee        != 0)) {
-		// remove_coinbase_kernel_flag test requires a plain kernel to have 0 fee
+		if   self.is_coinbase()      && self.fee         != 0
```

Comment:
rustfmt is going to clean your nicely aligned whitespace up next time somebody runs it. So may as well clean it up now.

Reply:
thx. will start running rustfmt before committing...